services/terraformer/templates/azure.tpl

```diff
@@ -103,7 +103,7 @@ resource "azurerm_subnet" "{{ $nodepool.Name }}_{{ $clusterHash }}_subnet" {
   name                 = "{{ $nodepool.Name }}_{{ $clusterHash }}_subnet"
   resource_group_name  = azurerm_resource_group.rg_{{ replaceAll $nodepool.Region " " "_"  }}.name
   virtual_network_name = azurerm_virtual_network.claudie_vn_{{ replaceAll $nodepool.Region " " "_" }}.name
-  address_prefixes     = ["{{getCIDR "10.0.0.0/24" 2 $i}}"]
+  address_prefixes     = ["{{index $.Metadata (printf "%s-subnet-cidr" $nodepool.Name) }}"]
```

Comment:
this was failing for me when I tried to test it, I actually had a comment about it, I see you fixed it. :rocket: 
when I run it now it succeeds.

```diff
-address_prefixes     = ["{{index $.Metadata(printf "%s-subnet-cidr" $nodepool.Name) }}"]
+address_prefixes     = ["{{index $.Metadata (printf "%s-subnet-cidr" $nodepool.Name) }}"]
```